xbmc/profiles/ProfileManager.cpp

```diff
@@ -423,12 +423,14 @@ void CProfileManager::FinalizeLoadProfile()
   // the startup window is considered part of the initialization as it most likely switches to the final window
   bool uiInitializationFinished = firstWindow != WINDOW_STARTUP_ANIM;
 
+  uiInitializationFinished = true;
```

Comment:
This looks strange as it unconditionally overwrites the init logic of this var (see line 424 and the comment in line 423)

Reply:
This was intentional (as a test). I didn't notice any reason why we really need this var; so set it as true before ripping anything.

Reply:
So you think @garbear messed it up as he introduced this code here?

Reply:
No -- I'm just not sure if it is needed still.